cr8/run_crate.py

```diff
@@ -187,20 +191,21 @@ def start(self):
         self.process = proc = self.enter_context(subprocess.Popen(
             self.cmd,
             stdin=subprocess.DEVNULL,
-            stdout=subprocess.PIPE,
+            stdout=subprocess.DEVNULL,
             stderr=subprocess.DEVNULL,
             env=self.env,
             universal_newlines=True
         ))
         log.info('PID: %s', proc.pid)
-        self.http_url = _get_http_uri(proc.stdout)
-        if not self.http_url:
-            log.warn("Couldn't get HTTP URL")
-            return
-
+        try:
+            self._obtain_http_url()
+        except NoHttpAddressAvailable as ex:
+            log.error(ex)
+            return False
```

Comment:
good point. had a version first which didn't raise an exception so I had to check for True/False if the node was started